src/main/java/io/fabric8/jenkins/openshiftsync/BuildConfigWatcher.java

```diff
@@ -94,6 +95,7 @@ public void doRun() {
         for(String namespace:namespaces) {
           try {
             logger.fine("listing BuildConfigs resources");
+            getOpenShiftClient().getConfiguration().setOauthToken(CredentialsUtils.getCurrentToken());
```

Comment:
this pattern seems pretty painful.  is there no way to set this up within getOpenShiftClient() without introducing a loop?